server/index.js

```diff
@@ -49,6 +49,18 @@ app.get('/api/products/:product_id', (req, res) => {
     });
 });
 
+app.get('/products/:product_id/styles', (req, res) => {
```

Comment:
I think the only difference in our two routes is that we're now using `/api/proucts/....` for our routes that hit the API.

Reply:
added /api/ to front of endpoint